src/backend/marsha/core/tests/test_lti.py

```diff
@@ -192,6 +192,31 @@ def test_lti_passport_consumer_site(self, mock_verify):
         self.assertEqual(lti.verify(), passport.consumer_site)
         self.assertEqual(mock_verify.call_count, 1)
 
+    @mock.patch.object(LTIOAuthServer, "verify_request", return_value=True)
+    def test_lti_passport_consumer_site_custom_show_download(self, mock_verify):
```

Comment:
Do we have the reverse test that proves that if the `show_download` flag is set to True, the video is created with `show_download` set to True ?

Reply:
No I created one test that randomly choose `True` or `False` and check that the value in the video object is the same in the consumer_site object. I should maybe create 2 tests ?

Reply:
I will create two tests